sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -1313,7 +1313,7 @@ describe("LocalWorkspace", () => {
 
     it("can import resources into a stack using resource definitions", async () => {
         const workDir = upath.joinSafe(__dirname, "data", "import");
-        const stackName = `int_test${getTestSuffix()}`;
+        const stackName = fullyQualifiedStackName(getTestOrg(), "node_test", `int_test${getTestSuffix()}`);
```

Comment:
I needed to specify an organisation explicitly for this test to pass.

Reply:
I'm surprised this is necessary now, but wasn't before.  Out of curiosity, what's the error if you don't do this?

Reply:
```
     CommandError: code: -2
 stdout:
 stderr: Command failed with exit code 255: pulumi stack init int_teste94f8441-6374-4fa3-80bd-3d5ff5bffb24 --non-interactive
error: could not create stack: [404] Not Found: Organization 'pulumi-test' not found
 err?: Error: Command failed with exit code 255: pulumi stack init int_teste94f8441-6374-4fa3-80bd-3d5ff5bffb24 --non-interactive
error: could not create stack: [404] Not Found: Organization 'pulumi-test' not found
```

I assume this is also because I have an access token?

Reply:
Ah hrm weird, I'm not sure where it gets `pulumi-test` from here if we're setting PULUMI_TEST_ORG :thinking: 

I'm fine with that change btw, was just curious where this is coming from.  But it's also not that important.